ignition_ros2_control/src/ignition_ros2_control_plugin.cpp

```diff
@@ -304,7 +310,7 @@ void IgnitionROS2ControlPlugin::Configure(
 
   if (!rclcpp::ok()) {
     rclcpp::init(static_cast<int>(argv.size()), argv.data());
-    this->dataPtr->node = rclcpp::Node::make_shared("ignition_ros2_control");
+    this->dataPtr->node = rclcpp::Node::make_shared("controller_manager_ignition_ros2_control");
```

Comment:
We should probably leave the option to set also the name of this node. This would be useful to run multi-robot simulation with multiple controller managers. (see my comment above)

Reply:
Can we also prefix this node with `controllerManagerPrefixNodeName`?